### PR TITLE
Dockeragent

### PR DESCRIPTION
**Pull request by Anjalikumari15**

### CODE REVIEW

**Review by yuktanandwani3 — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -1,32 +1,62 @@
...
+        }
+
+      }
       steps {
         sh 'mvn package -DskipTests'
         archiveArtifacts 'target/*.war'
       }
     }
 
+    stage('Docker build and publish') {
```
Comment (nupur-bansal) [inline Jenkinsfile:41]: You have not added agent for this stage ?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,32 +1,62 @@
...
+        }
+
+      }
       steps {
         sh 'mvn clean test'
       }
     }
 
     stage('package') {
+      agent {
```
Comment (nupur-bansal) [inline Jenkinsfile:29]: Also this needs to be run only on master

Comment (Anjalikumari15) [reply Jenkinsfile:29]: Done

File: Jenkinsfile (hunk 1 above)
Comment (Anjalikumari15) [reply Jenkinsfile:44]: Added it

**Review by nupur-bansal — APPROVED** [review]

**Pull request merged**